easybuild/easyconfigs/l/libvdwxc/libvdwxc-0.3.0-foss-2017b.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'libvdwxc'
+version = '0.3.0'
+
+homepage = 'http://libvdwxc.readthedocs.io/en/latest/index.html'
+description = """
+libvdwxc is a general library for evaluating energy and potential for
+exchange-correlation (XC) functionals from the vdW-DF family that can be
+used with various of density functional theory (DFT) codes. This work
+was inspired by success of libXC, a library for local and semilocal XC
+functionals. At the moment, libvdwxc provides access to the DF1, DF2,
+and CX functionals and interfaces for GPAW and Octopus. The library has
+been tested with respect to the S22 test set, various bulk properties of
+metals and semiconductors, and surface energies.
+"""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://gitlab.com/%(name)s/%(name)s/repository/%(version)s/']
+sources = [{'download_filename': 'archive.tar.bz2', 'filename': '%(name)s-%(version)s.tar.bz2'}]
```

Comment:
Hmm, better download a particular commit here or something, since this won't be correct for very long?

Reply:
Thats the 0.3.0 tag so should stay the same i hope.

Reply:
Ah, yes, good point, I missed that, my bad.